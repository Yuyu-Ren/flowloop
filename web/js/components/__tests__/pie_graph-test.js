/* eslint-env jest */

import PieGraph from '../pie_graph'

import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

describe('PieGraph', () => {
  it('exists', () => {
    expect(PieGraph).toBeTruthy()
  })

  it('renders', () => {
    const comp = <PieGraph steps={12} progress={0.6} color='red' />
    const wrapper = shallow(comp)
    expect(shallowToJson(wrapper)).toMatchSnapshot()
  })
})
