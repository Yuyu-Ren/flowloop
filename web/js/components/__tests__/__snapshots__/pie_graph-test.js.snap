// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PieGraph renders 1`] = `
<div
  className="pie-graph"
>
  <span
    className="fill"
    style={
      Object {
        "background": "red",
      }
    }
  />
  <span
    className="piece -fill"
    style={
      Object {
        "background": "red",
        "transform": "rotate(30deg)",
      }
    }
  />
  <span
    className="center"
  />
</div>
`;
